frontend/server/src/Controllers/Problem.php

```diff
@@ -4742,6 +4743,36 @@ public static function getProblemListForSmarty(
         ];
     }
 
+    /**
+     *
+     * @return array{smartyProperties: array{payload: ProblemListCollection, title: string}, entrypoint: string}
+     *
+     * @omegaup-request-param string $problemLevelTag
```

Comment:
Ups, error mío. Esto ya cambió por 
```suggestion
     * @omegaup-request-param string $problem_level_tag
```